lib/surface_bulma/dropdown/link_item.ex

```diff
@@ -0,0 +1,21 @@
+defmodule SurfaceBulma.Dropdown.LinkItem do
+  use Surface.Component, slot: "menu_items"
+
+  @moduledoc """
+  An item in the dropdown menu
+  """
+
+  @doc "Link target"
+  prop to, :string, required: true
```

Comment:
This seems to be an unused prop. Also, we might need to think about different types of links: normal, live_redirect, live_push

Reply:
I agree. What do you think about creating three versions of this component (one for each of those options)?

Reply:
I am currently experimenting with wrapping the Surface.LiveRedirect and LivePatch components with a `Context` that will fetch the appropriate class for the `a` tag. This way, the user can just use a LiveRedirect in different situations without having to worry about the right class and we do not have to create a new type of link component for each parent component that might contain a link. The Navbar.Item is now a bit hacky because of this, where an item can be 

An example would be:
```jsx
<Dropdown>
  <LiveRedirect to="/">Home</LiveRedirect>
</Navbar>
```
With the resulting markup then being:
```html
<nav class="navbar">
  <a class="navbar-item" href="/">Home</a>
</nav>
```
The implementation is something along the lines of:
```elixir
defmodule SurfaceBulma.LiveRedirect do
  def render(assigns) do
    ~F"""
      <Context get={SurfaceBulma.Link, link_class: class}>
        <Surface.LiveRedirect class={@class || class} ... />
      </Context>
    """
  end
end

defmodule ComponentThatCanHaveLinkWithSpecialClass do
  slot default # Links can be included here
  def render(assigns) do
    ~F"""
      <Context put={SurfaceBulma.Link, link_class: "special_class"}>
        <#slot />
      </Context>
    """
  end
end
```
WDYT?

Reply:
I asked for help accomplishing this very idea on the Slack channel but didn't get a great answer. I like this train of thought but I'm not sure how to finish it off the top of my head. How should we handle proceeding on my end so that you are able to modify?

Reply:
I saw your question in the Slack channel and at the moment I could really relate to your issue. Unfortunately I did not think of this approach until yesterday...

I think it would be good if we keep your PR as it is at the moment and merge it. Later, when I finish the proposed work as mentioned above I will update the `Dropdown` and create a new release (which is long overdue).

Thank you again for the work you put into this PR!

Reply:
I understand completely. I look forward to seeing your solution/progress on that issue.